tests/test_filter.py

```diff
@@ -2,7 +2,7 @@
 from unittest import mock
 import numpy as np
 import numpy.testing as npt
-import pyfar.dsp.filter as hpfilter
+import pyfar.dsp.classes as hpfilter
```

Comment:
hpfilter sounds like haiopy - how about pfilt or something like that?

Reply:
true, it's only in the tests though.